src/films/hdrfilm.cpp

```diff
@@ -310,8 +310,6 @@ class HDRFilm final : public Film<Float, Spectrum> {
                         dest_field.name = m_channels[i];
                         break;
                 }
-
-                source_field.name = m_channels[i];
```

Comment:
Curious about this deletion.

Reply:
The list of channel names can be passed to the `Bitmap` constructor (see L259).

Reply:
ok!